validator/keymanager/v2/direct/import.go

```diff
@@ -39,6 +39,15 @@ func (dr *Keymanager) ImportKeystores(
 			return errors.Wrap(err, "could not add to progress bar")
 		}
 	}
+	foundKey := map[string]bool{}
+	var strKey string
+	for j := range pubKeys {
```

Comment:
```suggestion
	for i := range pubKeys {
```

Reply:
Thanks again, I just pushed these fixes.  I'm just getting used to go that I have neven seen. I feel like I'm wasting a bit of your time. I have a few issues here that I can submit PRs, but if you're wasting more energy in reviewing them than in solving them yourselves, it's probably detrimental.